### PR TITLE
Improve metric threshold

### LINKED ISSUE #2396: New Feature Enhancement: Maximum option for calculation of metrics display
It would be great if the "calculation of metrics" had an option for "max value". This would be helpful when attempting to report on the maximum or high water mark of a metric over a period of time.

image

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #2396

---

@GrahamCampbell and I have previous discussed this.

### CODE REVIEW

File: app/Models/Metric.php [hunk 1]
```diff
@@ -201,6 +210,26 @@ public function getShouldDisplayAttribute()
...
+     * @throws \AltThree\Validator\ValidationException
+     *
+     * @return void
+     */
+    public function validate()
+    {
+        $messages = [];
+
+        if (60 % $this->threshold !== 0) {
+            $messages[] = 'Threshold must be divisible by 60.';
```
Comment (GrahamCampbell) [inline app/Models/Metric.php]: No, the threshold must **divide** 60. Not the same thing.

Comment (jbrooksuk) [inline app/Models/Metric.php]: Fixed!

File: app/Models/Metric.php [hunk 2]
```diff
@@ -221,7 +221,7 @@ public function validate()
         $messages = [];
 
         if (60 % $this->threshold !== 0) {
-            $messages[] = 'Threshold must be divisible by 60.';
+            $messages[] = 'Threshold must divide by 60.';
```
Comment (GrahamCampbell) [inline app/Models/Metric.php]: Just `divide 60`, not by.

Comment (jbrooksuk) [inline app/Models/Metric.php]: Not `into 60`? The wording doesn't make sense haha.

Comment (GrahamCampbell) [inline app/Models/Metric.php]: We'd say `The threshold must divide 60.`, or we could notate this `t | 60` for `t divides 60`.

Comment (jbrooksuk) [inline app/Models/Metric.php]: You're a mathematician though 😂

**Pull request merged**